src/App.vue

```diff
@@ -17,3 +17,17 @@ export default defineComponent({
   }
 });
 </script>
+
+<style>
+.modal-authen-inner-container-width {
```

Comment:
why make it global variable?

Reply:
Because you say so.

Reply:
yes, for reusable. Its name don't sound reusable